test/unit/bucket/test_bucket.py

```diff
@@ -1195,11 +1208,60 @@ def test_upload_local_file(self):
             write_file(path, data)
             file_info = self.bucket.upload_local_file(path, 'file1')
             self._check_file_contents('file1', data)
+            self._check_large_file_sha1('file1', None)
             self.assertTrue(isinstance(file_info, VFileVersionInfo))
             self.assertEqual(file_info.server_side_encryption, SSE_NONE)
             print(file_info.as_dict())
             self.assertEqual(file_info.as_dict()['serverSideEncryption'], {'mode': 'none'})
 
+    @pytest.mark.apiver(from_ver=2)
```

Comment:
This test would probably work OK with version 1, but in that version the minimum size of a file part is hardcoded to 5 MB.  The simulator setting is ignored...

Any suggestions how to get around this?

Reply:
ignore it - we don't want to spend too much time supporting new features on v1